theme/common.less

```diff
@@ -1805,6 +1815,15 @@ div.simframe.ui.embed {
                 justify-self: start;
                 max-width: 100%;
             }
+
+            div.row2 {
```

Comment:
I don't suppose there is a more descriptive name we could use?

Reply:
That's fair :)